hera_pspec/testing.py

```diff
@@ -401,22 +527,87 @@ def gauss_cov_fg(cov_amp, cov_length_scale, freqs, Ntimes=100, constant_in_time=
     # generate a random process from a Gaussian covariance
     Nfreqs = len(freqs)
 
-    C = cov_amp * np.exp(-((freqs[:, None] - freqs[None, :]) / cov_length_scale)**2)  # a covariance model
+    C = cov_amp * np.exp(
+        -(((freqs[:, None] - freqs[None, :]) / cov_length_scale) ** 2)
+    )  # a covariance model
 
     if constant_in_time:
         s = stats.multivariate_normal.rvs(np.zeros(Nfreqs), C, 2).reshape(2, 1, Nfreqs)
         s = s[0] + 1j * s[1]
         s = np.repeat(s, Ntimes, axis=0)
 
     else:
-        s = stats.multivariate_normal.rvs(np.zeros(Nfreqs), C, 2 * Ntimes).reshape(2, Ntimes, Nfreqs)
+        s = stats.multivariate_normal.rvs(np.zeros(Nfreqs), C, 2 * Ntimes).reshape(
+            2, Ntimes, Nfreqs
+        )
         s = s[0] + 1j * s[1]
 
     return s
 
 
-def sky_noise_sim(data, beam, cov_amp=1000, cov_length_scale=10, constant_per_bl=True,
-                  constant_in_time=True, bl_loop_seed=None, divide_by_nsamp=False):
+def sky_noise_jy_autos(lsts, freqs, autovis, omega_p, integration_time, channel_width=None, Trx=0.0):
+    """Make a noise realization for a given auto-visibility level and beam.
+
+    This is a simple replacement for ``hera_sim.noise.sky_noise_jy``.
+
+    Parameters
+    ----------
+    lsts : array_like
+        LSTs at which to compute the sky noise.
+    freqs : array_like
+        Frequencies at which to compute the sky noise, in Hz.
+    autovis : float
+        Autocorrelation visibility amplitude, in Jy.
+    omega_p : callable or array_like
+        A function of frequency giving the integrated beam area.
+    integration_time : float, optional
+        Integration time in seconds. By default, use the average difference
+        between given LSTs.
+    channel_width : float, optional
+        Channel width in Hz, by default the mean difference between frequencies.
+    Trx : float, optional
+        Receiver temperature, in K.
+
+    Returns
+    -------
+    noise : ndarray
+        2D array of white noise in LST/freq.
+    """
+    # Beam solid angle
+    if callable(omega_p):
+        omega_p = omega_p(freqs)
+    
```

Comment:
Add check that if not callable, then array of length same as freqs. Maybe assert freqs is array too

Reply:
Done!